devel/query.lua

```diff
@@ -169,12 +169,19 @@ function main()
     query(selection, path_info, args.search, path_info)
 end
 
+local eval_env = utils.df_shortcut_env()
+function eval(s)
+    local f, err = load("return " .. s, "expression", "t", eval_env)
+    if err then qerror(err) end
+    return f()
+end
```

Comment:
Made an issue for the to-do we discussed on Discord around refactoring this: https://github.com/DFHack/dfhack/issues/2928